src/components/ProductPage/ProductTokenStats.tsx

```diff
@@ -11,13 +12,15 @@ interface ProductTokenStatsProps {
   fees?: {
     streamingFee: string
   }
+  nav: number
```

Comment:
I think (in general) we should use longer, more descriptive variable names over short ones where possible. i think a lot of people might confuse this as having to do with navigation. just `netAssetValue` would be helpful in reading the code.

Reply:
I'll change this everywhere